verification/serverless.yml

```diff
@@ -91,9 +91,9 @@ functions:
               - X-Amz-Security-Token
               - X-Amz-User-Agent
 
-  duns-callback:
+  callback:
```

Comment:
The function name is generic. Previously we had duns-callback, we need a similar name.

Reply:
we will use the same callback for both